packages/near-fast-auth-signer-e2e-tests/tests/signMultiChain/sign-multi-chain.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { KeyPair } from '@near-js/crypto';
+import { expect, Page, test } from '@playwright/test';
+
+import {
+  receivingAddresses, getFastAuthIframe
+} from '../../utils/constants';
+import { overridePasskeyFunctions } from '../../utils/passkeys';
+import { isWalletSelectorLoaded } from '../../utils/walletSelector';
+import SignMultiChain from '../models/SignMultiChain';
+
+let page: Page;
+let signMultiChain: SignMultiChain;
+
+const userFAK = process.env.MULTICHAIN_TEST_ACCOUNT_FAK;
+const accountId = process.env.MULTICHAIN_TEST_ACCOUNT_ID;
+
+const fakKeyPair = KeyPair.fromString(userFAK);
+
+const isAuthenticated = async (loggedIn: boolean) => {
+  if (!page) return;
+  if (loggedIn) {
+    await overridePasskeyFunctions(page, {
+      creationKeypair:  fakKeyPair,
+      retrievalKeypair: fakKeyPair
+    });
+    return;
+  }
+  await overridePasskeyFunctions(page, {
+    creationKeypair:  KeyPair.fromRandom('ed25519'),
+    retrievalKeypair: KeyPair.fromRandom('ed25519')
+  });
+};
+
+test.describe('Sign MultiChain', () => {
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    signMultiChain = new SignMultiChain(page);
+    await page.goto('/');
+    await page.waitForLoadState('domcontentloaded');
+    // Disable pointer events that will interrupt click event on "Approve button"
```

Comment:
Explain why this is necessary. This doesn't seems to be something that the user can do on his side while using the app

Reply:
`webpack-dev-server` adds a hidden iframe to projects (for hot reloading), that iframe stays on top of the page components and causes event propagation, which means a click event by playwright will have multiple effects, so it needs to be disbaled.

This can't happen in `prod` for ideal user because `webpack-dev-server` is used for development.

Reply:
Got it, please add this explanation on the code so it's easier for us to know why we are disabling it.

Reply:
Done